vic/drivers/shared_all/src/vic_history.c

```diff
@@ -30,7 +30,7 @@
  * @brief    This routine creates the list of output data.
  *****************************************************************************/
 void
-alloc_out_data(size_t     ngridcells,
+alloc_out_data(size_t    ngridcells,
```

Comment:
make sure you run all the code through `uncrustify` (see: https://github.com/UW-Hydro/VIC/tree/master/tools/code_format)